### DISCUSSION TITLE
Rücksetzung Tagesertrag

### DISCUSSION THREAD

**Discussion by peff74** [hoylabs/OpenDTU-OnBattery#207] [Q&A] [answered]
Noch eine Frage^^
Gibt es irgendwo eine Einstellung um den Tagesertrag zurück zusetzten?
Konnte nichts finden und heute werden die Daten auf die von gestern addiert?

Danke
Peff

**Comment by helgeerbe** [comment]
Hi @peff74,

nein gibt es nicht. Das Problem ist, dass der Inverter den Tageszähler zurücksetzt, wenn kein Strom von den Panels kommt. Wenn du nun also eine Batterie dran hast, wird der Tageszähler nicht mehr genullt. Du musst dir ggf. dein eigenes Dashboard in Homeassistant, IoBroker, node-red, Grafana bauen.

**Reply by peff74** [reply]
@helgeerbe 
Ok ich dachte es gäbe da was (MQTT) um den Inverter 'anzutrigger' die Tageswerte zu Nullen.
Meine ich hätte da was gelesen bei Ahoy

**Reply by helgeerbe** [reply, marked as answer]
Du könntest um Mitternacht den Inverter neu starten. Damit müssten die Werte auch genullt werden. Aber der Tageszähler ist ehh nicht zu gebrauchen. Wenn du den Inverter mal abklemmst, stimmt es eh nicht mehr.  Besser ist den totalen Wert um Mitternacht zu nehmen und dass Delta dann als Tagesbezug zu verwenden. Wert total - Wert um Mitternacht

**Reply by peff74** [reply]
Ok. Gefressen und akzeptiert ^^
